subsys/usb/class/cdc_acm.c

```diff
@@ -425,18 +421,27 @@ static void cdc_acm_do_cb(struct cdc_acm_dev_data_t *dev_data,
 		dev_data->tx_ready = true;
 		dev_data->tx_irq_ena = true;
 		dev_data->rx_irq_ena = true;
+		dev_data->usb_status = status;
 		LOG_INF("USB device configured");
 		break;
 	case USB_DC_DISCONNECTED:
 		LOG_INF("USB device disconnected");
 		cdc_acm_reset_port(dev_data);
+		dev_data->usb_status = status;
 		break;
 	case USB_DC_SUSPEND:
-		LOG_INF("USB device suspended");
+		LOG_INF("CDC ACM suspended");
```

Comment:
Just a minor note. Let's keep `USB device` or change to `CDC ACM` everywhere in `cdc_acm_do_cb`.

Reply:
I would like to remove both USB device and CDC ACM because every log line is prefixed with `usb_cdc_acm: .... `

Reply:
done